be/src/storage/olap_server.cpp

```diff
@@ -462,6 +467,15 @@ void* StorageEngine::_local_pk_index_shard_data_gc_thread_callback(void* arg) {
                         lake_update_manager->unlock_pk_index_shard(id);
                     }
                 }
+                auto mtime_or = FileSystem::Default()->get_file_modified_time(tablet_pk_path);
```

Comment:
If l0 file in pk path is continue append, will path's modified time change?

Reply:
> I have tested in linux server. I created a directory and touch a file in that directory. If the file is modified, the modified time of its parent directory is also changed.

This should not append, at lease should not happen on ext[234], the directory modification time doesn't change if a file is modified.

Reply:
Did you actually delete the original file and then create a file with the same name when you tested it?